.env.example

```diff
@@ -24,10 +24,14 @@ SERVER_PORT=9090
 SERVER_ADDRESS=hapi
 HASURA_URL=http://hasura:8080/v1/graphql
 HASURA_ADMIN_SECRET=myadminsecretkey
-EOS_API_URL=https://jungle2.eosio.cr
-EOS_API_CHAIN_ID=e70aaab8997e1dfce58fbfac80cbbb8fecec7b99cf982a9444273cbc64c41473
+COMMUNITY_ASSET=BLOOD
+LIFEBANKCODE_CONTRACTNAME=lifebankcode
+CONSENT2LIFE_CONTRACTNAME=consent2life
+LIFEBANKCOIN_CONTRACTNAME=lifebankcoin
+EOS_API_URL=https://jungle.eosio.cr
+EOS_API_CHAIN_ID=2a02a0053e5a8cf73a56ba0fda11e4d92e0238a4a2aa74fccf46d5a910746840
 EOS_BASE_ACCOUNT=lbacccreator
-EOS_BASE_ACCOUNT_PASSWORD=PW5Huw9Sk2iBrbHEQ37iZrPAQof7oeSZVtDsgBuiqdDouTDUnywn1
+EOS_BASE_ACCOUNT_PASSWORD=PW5KYpFXvpQ9zTB9zB5NwzorL3W1Rz1K2YFXFV4LjJCRphfo7W1oF
```

Comment:
is it safe to expose this value ?

Reply:
it's the password for the local wallet should not be a problem but could be better keep it empty in the env.example file because the password will be different for each local environment